### DISCUSSION TITLE
MacOS workflow editor: Submit to Testflight" checkbox

### DISCUSSION THREAD

**Discussion by jmgeffroy** [codemagic-ci-cd/codemagic-docs#2731] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Create a workflow to publish the MacOS app on Testflight / AppstoreConnect.

### Expected results

I expect to see the same "Submit to Testflight" checkbox in the MacOS workflow editor as present in the iOS workflow editor.

### Actual results

No "Submit to Testflight" checkbox in the MacOS workflow.

**Comment by dtrdic** [comment]
Hey @jmgeffroy that's because the build will be automatically submitted to Testflight. If you need your build to be automatically sent for review then you would have to use yaml configuration and set this flag in the publishing section `submit_to_app_store: true`

**Comment by dtrdic** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
